components/brave_wallet/browser/brave_wallet_prefs.cc

```diff
@@ -183,6 +183,10 @@ void RegisterProfilePrefsForMigration(
   // Added 03/2023
   registry->RegisterBooleanPref(kBraveWalletUserAssetsAddIsERC1155Migrated,
                                 false);
+
+  // Added 03/2023
```

Comment:
maybe we should updated to 04/2023 now to be accurate.

Reply:
86724eb159af5224c1a67eb38285a820a52e45b3